src/MOL_discretization.jl

```diff
@@ -2,8 +2,21 @@ using ModelingToolkit: operation
 # Method of lines discretization scheme
 struct MOLFiniteDifference{T} <: DiffEqBase.AbstractDiscretization
     dxs::T
-    order::Int
-    MOLFiniteDifference(args...;order=2) = new{typeof(args[1])}(args[1],order)
+    upwind_order::Int
+    centered_order::Int
+end
+
+# Constructors. If no order is specified, both upwind and centered differences will be 2nd order

```

Comment:
I think it's natural to default to 2nd order central and 1st order upwinding?

Reply:
Yep you're right. I've been swamped with work but will try to push this along soon.